pyeudiw/satosa/default/request_handler.py

```diff
@@ -25,6 +25,35 @@
 
 
 class DefaultRequestHandler(RequestHandlerInterface, BackendTrust):
+    def _handle_trustability(self, context: Context, vp: Vp) -> bool:
+        try:
+            # establish the trust with the issuer of the credential by checking it to the revocation
+            # inspect VP's iss or trust_chain if available or x5c if available
+            # TODO: X.509 as alternative to Federation
+
+            # for each single vp token, take the credential within it, use cnf.jwk to validate the vp token signature -> if not exception
+            # establish the trust to each credential issuer
+            tchelper = self._validate_trust(context, vp.payload['vp'])
+
+            if not tchelper.is_trusted:
+                return self._handle_400(context, f"Trust Evaluation failed for {tchelper.entity_id}")
+
+            # TODO: generalyze also for x509
+            credential_jwks = tchelper.get_trusted_jwks(
```

Comment:
trust and credentials are different things, then this method is related to credential evaluation and trust evaluation to their issuers

Reply:
Could "_handle_credential_trust" be a good name?

Reply:
yeah, we can do that